src/components/header.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import './style.css';
+
+function Header() {
+  return (
+    <div>
+      <header>
+        <h2 className="bookstore">Bookstore CMS</h2>
+        <nav>
+          <a href="/" className="books">
+            Books
+          </a>
+          <a href="/categories" className="categories">
+            Categories
+          </a>
```

Comment:
- [OPTIONAL] This was not pointed out by the previous review, but as part of the requirement, you installed react-router, therefore the navigation should also be handled by react-router, Currently clicking on the links refreshes the page to load and that's not great in a SINGLE PAGE APPLICATION. I'll suggest you import `Link or useNavigate` from `'react-router-dom';` and use it to handle the navigation. :+1: :100: